src/components/Profile.js

```diff
@@ -85,17 +87,17 @@ class Profile extends Component<Props> {
               />
             }
           </h1>
-          {/* // TODO make below work, pass in correct props
           {
             this.props.trustedByUids !== undefined &&
+            // TODO pass in correct props. Should MemberRelations mapStateToProps be in redux?
             <TrustLevel
               ownProfile={ownProfile}
-              trustLevel={3}
+              trustLevel={memberDoc.get('invite_confirmed') ? 3 : 0}
```

Comment:
This will be accurate today because all of the `'invite_confirmed'` users joined in January and are trust level 3. As soon as we start adding new blocks again we will need to have this actually working